repo/packages/C/chronos/0/config.json

```diff
@@ -115,7 +115,7 @@
       "description": "Memory (MB) to allocate to each Chronos instance.",
       "type": "number",
       "minimum": 512.0,
-      "default": 700.0
+      "default": 1024.0
```

Comment:
Heap is actually set separately in our packaging currently. It would be ideal to be able to pass it as well.